sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/early_termination_policy.py

```diff
@@ -105,7 +105,7 @@ def __init__(
         self,
         *,
         delay_evaluation: int = 0,
-        evaluation_interval: int = 0,
+        evaluation_interval: int = 1,
```

Comment:
only need to change default for this type of policy?

Reply:
Will be changing for other policies in separate PR.